docs/advanced/payments/adyen.mdx

```diff
@@ -54,6 +54,9 @@ to integrate with the Adyen Magento module please refer to
 - `FRONT_COMMERCE_ADYEN_STORE_PAYMENT_DETAILS` Boolean (more on
   [Adyen's Stored Payment Methods](#enable-adyens-stored-payment-methods-optional-1)).
   If you want to enable Adyen's Stored Payment Methods (Tokenization).
+- `FRONT_COMMERCE_ADYEN_MAGENTO_MODULE_VERSION` The version of the Magento2
+  Adyen module you use. You can find it in your magento backend, in
+  `Stores > Configuration > Sales > Payment Methods > Adyen Payments > Documentation and Support`.
```

Comment:
this is under `As a Front-Commerce payment method` while this is for the case where the Magento2 module is used.

also, while writing that, I realize that this environment variable is probably required also in the case where embedded payment is used and that should not be the case. 

and maybe you can add there that we only support the module in version 7 and 8 ?

Reply:
> this is under As a Front-Commerce payment method while this is for the case where the Magento2 module is used.

Indeed ! Will update 👍 

> also, while writing that, I realize that this environment variable is probably required also in the case where embedded payment is used and that should not be the case.

As discussed, will add a second config provider to complement the existing one with the new env variable

> and maybe you can add there that we only support the module in version 7 and 8 ?

Will do !